tests/Unit/Managers/ProcessManagerTest.php

```diff
@@ -0,0 +1,27 @@
+<?php declare(strict_types = 1);
+
+namespace Churn\Tests\Unit\Managers;
+
+use Churn\Configuration\Config;
```

Comment:
```suggestion
use Churn\Collections\FileCollection;
use Churn\Configuration\Config;
```

The test fails because of this missing import:
```
1) Churn\Tests\Unit\Managers\ProcessManagerTest::it_returns_collection_with_same_count_as_number_of_parallel_jobs
Error: Class 'Churn\Tests\Unit\Managers\FileCollection' not found
/home/travis/build/bmitch/churn-php/tests/Unit/Managers/ProcessManagerTest.php:24
```